src/components/common/TmSessionExplore.vue

```diff
@@ -104,7 +104,14 @@ export default {
     error: ``
   }),
   computed: {
-    ...mapState([`session`])
+    ...mapState([`session`]),
+    ...mapGetters([`network`]),
+    filteredAddresses() {
+      const networkPrefix = this.network.split(`-`)[0]
```

Comment:
this doesn't work reliably. It only be chance works with the current networks. What about Regen i.e.. Their address prefix is `xrn:`. You need to do an address lookup.

Btw.: The address lookup is needed in several cases. We should refactor it and put it someplace. Maybe the store?

Reply:
Wow, you are right! How could I miss the regen addresses?

OK, let me refactor this. Yes, maybe it could be one of those `store.dispatch` functions

Reply:
I found we already this address lookup in place. It exists in the `config.js` file and it is called `bech32Prefixes`

Reply:
Needs to move out of there: #3402

Reply:
But we can do that in another PR